src/modules/tests/test-data/test-data.actions.ts

```diff
@@ -33,11 +35,13 @@ export const CONTROLLED_STOP_ADD_DRIVING_FAULT = '[ControlledStop] Add Driving F
 export const CONTROLLED_STOP_ADD_SERIOUS_FAULT = '[ControlledStop] Add Serious Fault';
 export const CONTROLLED_STOP_ADD_DANGEROUS_FAULT = '[ControlledStop] Add Dangerous Fault';
 export const CONTROLLED_STOP_REMOVE_FAULT = '[ControlledStop] Remove Fault';
+export const CONTROLLED_STOP_ADD_COMMENT = '[ControlledStop] Add Comment';
```

Comment:
The name here isn't quite consistent with the other new actions you added. `ADD_MANOEUVRE_COMMENT` vs `CONTROLLED_STOP_ADD_COMMENT`.

Reply:
Will rename to be consistent